components/resources/ipfs_strings.grdp

```diff
@@ -1,44 +1,49 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <grit-part>
   <if expr="enable_ipfs">
-    <message name="IDS_IPFS_CONNECTED_PEERS_TITLE" desc="Number of peers connected">Connected peers:</message>
-    <message name="IDS_IPFS_ADDRESSES_CONFIG_TITLE" desc="Title of addresses config section">Addresses</message>
-    <message name="IDS_IPFS_REPO_STATS_TITLE" desc="Title of repo stats section">Repo Stats</message>
-    <message name="IDS_IPFS_NODE_INFO_TITLE" desc="Title of node info section">Node info</message>
+    <if expr="enable_ipfs_internals_webui">
```

Comment:
Below are the strings from this section:
https://github.com/brave/brave-core/blob/master/browser/ui/webui/brave_webui_source.cc#L713